human_analysis/nbconverted/1_process_recount2_data.py

```diff
@@ -0,0 +1,389 @@
+
+# coding: utf-8
+
+# # Process recount2 data
+# This notebook does the following:
+# 
+# 1. Selects template experiment
+# 2. Downloads subset of recount2 data, including the template experiment (50 random experiments + 1 template experiment)
+# 3. Train VAE on subset of recount2 data
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('load_ext', 'rpy2.ipython')
+get_ipython().run_line_magic('autoreload', '2')
+
+import os
+import sys
+import pandas as pd
+import numpy as np
+import random
+import rpy2
+import seaborn as sns
+from sklearn import preprocessing
+import pickle
+
+sys.path.append("../")
```

Comment:
If you add a setup.py file to your repo, you can install the repo as a module and not have to manually set the path to your python functions. Probably would be the subject of a different PR if you choose to do so though